### PR TITLE
feature: refactor exec to enable configure stdin

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: YaoZengzeng <yaozengzeng@zju.edu.cn>

### Ⅰ. Describe what this PR did

Before this PR, we open the stdin of container IO by default.

Actually it's wrong, we should close it by default. 

Unless user specify like `pouch run -i ....`, then we should open it.

### CODE REVIEW

**Comment by YaoZengzeng** [comment]
@allencloud PTAL

**Comment by allencloud** [comment]
This is a pr which seems to affect the ctrd package. Could you help to review this? @HusterWan

File: daemon/containerio/cio.go [hunk 1]
```diff
@@ -0,0 +1,202 @@
+package containerio
```
Comment (HusterWan) [inline daemon/containerio/cio.go]: why we just copy containerd's Fifo io code to pouch, can we reuse containerd's code ? :)

Comment (YaoZengzeng) [reply daemon/containerio/cio.go]: @HusterWan Actually the code in containerd can't meet our need, we need to some modifications :)

**Comment by HusterWan** [comment]
LGTM

**Pull request merged**